runtime/layers/fpm-dev/Dockerfile

```diff
@@ -2,5 +2,10 @@ ARG LAYER_IMAGE
 FROM $LAYER_IMAGE
 
 COPY ./php-fpm.conf /opt/bref/etc/php-fpm.conf
+
+RUN if [ "$LAYER_IMAGE" = "php-73-fpm" ] ||  [ "$LAYER_IMAGE" = "php-73" ]; \
+    then echo "decorate_workers_output not supported"; \
+    else echo "decorate_workers_output = no" >> /opt/bref/etc/php-fpm.conf; fi
```

Comment:
Should it be the other way around? If 72 then "not supported", else we set the flag.

Reply:
true. It seems that when it's running the build image $LAYER_IMAGE is not available or empty 😩